pkgs/servers/irc/inspircd/default.nix

```diff
@@ -0,0 +1,192 @@
+let
+  # instead of inventing a license compatibility
+  # predicate, let's just whitelist licenses known
+  # to be compatible which are used by actual libcs
+  # in nixpkgs
```

Comment:
this comment as the first thing is kinda confusing, maybe add a comment above that about why it’s necessary in the first place.